docs/server/installation.rst

```diff
@@ -388,55 +232,23 @@ software. To run it, just issue:
 
    (env) $ pootle start
 
-
```

Comment:
hmm - having just updated quite a lot of docs my preference is defo only using 2 spaces to separate sections - its hard to see otherwise